core/string/ustring.h

```diff
@@ -39,6 +39,8 @@
 #include "core/typedefs.h"
 #include "core/variant/array.h"
 
+#include <utility>
```

Comment:
Too bad that C++ committee added move in the library and not the language itself. This is included a lot, _could_ have a small impact on compilation times.

Reply:
This could be replaced with our own macro theoretically, AFAIK `std::move` is just:
```cpp
static_cast<typename std::remove_reference<T>::type&&>(t)
```

Reply:
Just need our own remove_reference, which is not too bad to write.

Reply:
We already use `type_traits` so that shouldn't be critical